src/test/java/catan/controllers/testcases/play/UseCardMonopolyTest.java

```diff
@@ -0,0 +1,420 @@
+package catan.controllers.testcases.play;
+
+import catan.controllers.ctf.Scenario;
+import catan.controllers.ctf.TestApplicationConfig;
+import catan.controllers.util.PlayTestUtil;
+import catan.services.util.random.RandomUtil;
+import catan.services.util.random.RandomUtilMock;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.boot.test.WebIntegrationTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import static catan.domain.model.dashboard.types.HexType.BRICK;
+import static catan.domain.model.dashboard.types.HexType.EMPTY;
+import static catan.domain.model.dashboard.types.HexType.SHEEP;
+import static catan.domain.model.dashboard.types.HexType.STONE;
+import static catan.domain.model.dashboard.types.HexType.WHEAT;
+import static catan.domain.model.dashboard.types.HexType.WOOD;
+import static catan.domain.model.game.types.DevelopmentCard.MONOPOLY;
+import static java.util.Arrays.asList;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+
+//@SpringApplicationConfiguration(classes = {TestApplicationConfig.class, RequestResponseLogger.class})  // if needed initial request and JSON response logging:
+//@SpringApplicationConfiguration(classes = TestApplicationConfig.class)
+@SpringApplicationConfiguration(classes = TestApplicationConfig.class)
+@WebIntegrationTest("server.port:8091")
+public class UseCardMonopolyTest extends PlayTestUtil {
```

Comment:
test case is missing:
should_successfully_get_quantity_of_taken_resources()
i'll add it